server/src/main/java/com/google/coffeehouse/servlets/CreateClubServlet.java

```diff
@@ -50,16 +51,19 @@
   /** Message to be logged when the body of the POST request does not have required fields. */
   public static final String LOG_INPUT_ERROR_MESSAGE =
       "Error with JSON input in CreateClubServlet: ";
+  private final StorageHandlerApi handler;
   private static IdentifierGenerator idGen;
   private static final Gson gson = new Gson();
 
   /** 
    * Overloaded constructor for dependency injection.
    * @param idGen the {@link IdentifierGenerator} that is used when constructing the Club
+   * @param handler the {@link StorageHandlerApi} that is used when saving the Club
```

Comment:
Think about switching the order of these @params, right now they imply the opposite order

Reply:
Got it!